sem_covid/services/dataset_pipelines/unified_dataset_pipeline.py

```diff
@@ -0,0 +1,292 @@
+EU_CELLAR_COUNTRY_NAME = "European Union"
+EU_CELLAR_PWDB_ACTORS = "EU (Council, EC, EP)"
+EU_CELLAR_RENAME_COLUMNS_MAPPING = {"category": "pwdb_category", "funding": "pwdb_funding",
+                                    "type_of_measure": "pwdb_type_of_measure",
+                                    "dates_document": "date"
+                                    }
+EU_CELLAR_SPECIFIC_COLUMNS = {"eu_cellar_subject_matter_labels": "subject_matter_labels",
+                              "eu_cellar_resource_type_labels": "resource_type_labels",
+                              "eu_cellar_directory_codes_labels": "directory_codes_labels",
+                              "eu_cellar_author_labels": "author_labels"}
+
+# EU_TIMELINE CONSTANTS
+EU_TIMELINE_CONTENT_COLUMNS = ["title", "abstract", "detail_content"]
+EU_TIMELINE_DOC_SOURCE = 'ds_eu_timeline'
+EU_TIMELINE_COUNTRY_NAME = "European Union"
+EU_TIMELINE_PWDB_ACTORS = "EU (Council, EC, EP)"
+EU_TIMELINE_RENAME_COLUMNS_MAPPING = {"category": "pwdb_category",
+                                      "funding": "pwdb_funding",
+                                      "type_of_measure": "pwdb_type_of_measure",
+                                      }
+EU_TIMELINE_SPECIFIC_COLUMNS = {'eu_timeline_topic': 'topics'}
+
+# IRELAND_TIMELINE CONSTANTS
+IRELAND_TIMELINE_CONTENT_COLUMNS = ["title", "content"]
+IRELAND_TIMELINE_DOC_SOURCE = 'ds_ireland_timeline'
+IRELAND_TIMELINE_COUNTRY_NAME = "Ireland"
+IRELAND_TIMELINE_PWDB_ACTORS = "National government"
+IRELAND_TIMELINE_RENAME_COLUMNS_MAPPING = {"category": "pwdb_category",
+                                           "funding": "pwdb_funding",
+                                           "type_of_measure": "pwdb_type_of_measure",
+                                           "published_date": "date"
+                                           }
+IRELAND_TIMELINE_SPECIFIC_COLUMNS = {"ireland_keyword": "keyword",
+                                     "ireland_department_data": "department_data",
+                                     "ireland_campaigns_links": "campaigns_links",
+                                     "ireland_page_type": "page_type"}
+
+
+def make_target_group_l1_column(dataset: pd.DataFrame):
+    """
+    This function will create the pwdb_target_group_l1 column for a dataset. The value will be the content merged from
+    workers,businesses, citizens column values that are first converted from binary to text.
+    """
+    for col in TARGET_GROUPS_L1:
+        dataset[col] = dataset[col].apply(lambda x: col if x == 1 else "")
+
+    dataset["pwdb_target_group_l1"] = dataset[TARGET_GROUPS_L1].apply(lambda row: ' '.join(row.values.astype(str)),
+                                                                      axis=1).apply(lambda x: x.split())
+
+
+def make_topic_embeddings_column(dataset: pd.DataFrame):
+    """
+    This function created the topic_embeddings column for a dataset
+    """
+    dataset["topic_embeddings"] = [[0] * 50] * len(dataset)
```

Comment:
This is what I derive from the internet discussions:
```
dataset["topic_embeddings"] = None
dataset["topic_embeddings"].apply(lambda x: [0]*50 )
```

https://stackoverflow.com/questions/24039023/add-column-with-constant-value-to-pandas-dataframe
```
# Wrong
df['new'] = [[]] * len(df)
# Right
df['new'] = [[] for _ in range(len(df))]
```